e2e/cypress/integration/scroll/channel_scroll_spec.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+    let otherUser;
+
+    beforeEach(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                otherUser = user2;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2378 Channel with only a few posts opens at the bottom', () => {
+        // # Post a starting message with user 1
+        cy.postMessage('This is the first post');
+
+        // # Post as few as messages so that scroll bar appears as channel name scroll hidden
+        Cypress._.times(20, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
+
+        // # Post the last message
+        cy.postMessage('This is the last post');
+
+        // # Reload the browser
+        cy.reload();
+
+        // * Verify that the top of the channel is scrolled past hidden
+        cy.findByText(`Beginning of ${testChannel.display_name}`).should('exist').and('not.be.visible');
+
+        // * Verify that the last message is visible implying that channel scrolled to bottom on reload
+        cy.findByText('This is the last post').should('exist').and('be.visible');
+    });
+
+    it('MM-T2382 Center channel scroll', () => {
+        // # Post a starting message with user 1
+        cy.postMessage('This is the first post');
+
+        // # Scroll to bottom of the channel
+        // ensureScrollable is false as we dont want assertion to fail when we are already at bottom
+        // and there is no more to scroll
+        cy.get('div.post-list__dynamic').should('be.visible').scrollTo('bottom', {ensureScrollable: false});
```

Comment:
This failed on me, see screenshot.
<img width="545" alt="Screen Shot 2020-09-03 at 2 42 07 PM" src="https://user-images.githubusercontent.com/5334504/92080172-ae941e80-edf3-11ea-9bfc-d54192def73a.png">

btw, I'm fine with your previous commit so I went ahead and remove this part. Hope you don't mind.
```suggestion
```